bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
+except ImportError:
+    pass
+
+from bankroll.model import Position, Trade
+
+ConvertibleModel = Union[Position, Trade]
+ColumnFunctions = List[Tuple[str, Callable[[Any], Any]]]
+
+
+def dataframeForModelObjects(
+    items: Sequence[ConvertibleModel]
```

Comment:
I _think_ everything you're doing is covered by the `Iterable` interface (since you don't iterate the collection twice):

```suggestion
    items: Iterable[ConvertibleModel]
```

This is a bit more general, and supports things like generator comprehensions: `dataframeForModelObjects((x.z for x in y))`

Reply:
Actually I do iterate twice, which is why I changed it to a `Sequence`